packages/keyutils/keyutils.spec

```diff
@@ -0,0 +1,85 @@
+Name: %{_cross_os}keyutils
+Version: 1.6.1
+Release: 1%{?dist}
+Summary: Linux Key Management Utilities
+License: GPLv2+ and LGPLv2+
+Url: http://people.redhat.com/~dhowells/keyutils/
+Source0: http://people.redhat.com/~dhowells/keyutils/keyutils-%{version}.tar.bz2
+Source1: keyutils-tmpfiles.conf
+Source2: request-key.conf
+
+BuildRequires: %{_cross_os}glibc-devel
+
+%description
+%{summary}.
+
+%package devel
+Summary: Development package for building Linux key management utilities
+
+%description devel
+%{summary}.
+
+%prep
+%setup -n keyutils-%{version} -q
+
+%global set_env \
+%set_cross_build_flags \\\
+export CC=%{_cross_target}-gcc \\\
+%{nil}
+
+%global keyutilsmake \
+%make_build \\\
+  NO_ARLIB=1 \\\
+  ETCDIR=%{_cross_sysconfdir} \\\
+  LIBDIR=%{_cross_libdir} \\\
+  USRLIBDIR=%{_cross_libdir} \\\
+  BINDIR=%{_cross_bindir} \\\
+  SBINDIR=%{_cross_sbindir} \\\
+  MANDIR=%{_cross_mandir} \\\
+  INCLUDEDIR=%{_cross_includedir} \\\
+  SHAREDIR=%{_cross_datadir}/keyutils \\\
+%{nil}
+
+
+%build
+%set_env
+%keyutilsmake \
+  RELEASE=.%{release} \
+  NO_GLIBC_KEYERR=1 \
+  CC="${CC}" \
+  CFLAGS="${CFLAGS}" \
+  LDFLAGS="${LDFLAGS}"
+
+%install
+%keyutilsmake \
+  DESTDIR=%{buildroot} \
+  install
+
+install -d %{buildroot}%{_cross_tmpfilesdir}
+install -d %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}
+
+install -p -m 0644 %{S:1} %{buildroot}%{_cross_tmpfilesdir}/keyutils.conf
+install -p -m 0644 %{S:2} %{buildroot}%{_cross_factorydir}%{_cross_sysconfdir}/request-key.conf
+
+%files
+%{_cross_attribution_file}
+%license LICENCE.GPL LICENCE.LGPL
+%{_cross_tmpfilesdir}/keyutils.conf
+%{_cross_bindir}/keyctl
+%{_cross_sbindir}/key.dns_resolver
+%{_cross_sbindir}/request-key
+%{_cross_datadir}/keyutils
+%{_cross_libdir}/libkeyutils.so.*
+
+%dir %{_cross_factorydir}
```

Comment:
You can omit this; I only use `%dir` to claim ownership of a directory when it's clearly tied to this package, usually because it's a subdirectory named for the software. Otherwise you end up with multiple packages owning the same directory, which isn't ideal.

```suggestion
```